app/views/shared/_contact_panels.html.erb

```diff
@@ -26,7 +26,7 @@
 </a>
 <% end %>
 
-<%= render 'shared/webchat_popup', locals: {clumps: clumps, chat_opening_hours: chat_opening_hours} %>
+<%= render 'shared/webchat_popup', locals: {clumps: clumps, chat_opening_hours: chat_opening_hours, pensions_opening_hours: pensions_opening_hours} %>
```

Comment:
I think using locals here is a good thing for clarity, even though we don't need to because chat_opening_hours and pensions_opening_hours are helper methods that available globally in all view.

If we're using them here though, we should be consistent and pass them into the webchat_popup_pensions partial as well.

Reply:
I wasn't too sure so I had removed the locals from the pensions partial. I've now put it back in, thanks!